reference/docs-conceptual/community/contributing/editorial-checklist.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Editorial checklist
+description: This is a summarized list of rules for editing PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Editor's checklist
+
+This is a summary of rules to apply when writing new or updating existing articles. See other
+articles in the Contributor's Guide for detailed explanations and examples of these rules.
+
+## Metadata
+
+- `ms.date`: must be in the format **MM/DD/YYYY**
+  - Change the date when there is a significant or factual update
+    - Reorganizing the article
+    - Fixing factual errors
+    - Adding new information
+  - Do not change the date if the update is insignificant
+    - Fixing typos and formatting
+- `title`: unique string of 43-59 chars including spaces
+  - Do not include site identifier (it is auto-generated)
+  - Use sentence case - capitalize only the first word and any proper nouns
+- `description`: 115-145 characters including spaces - this abstract displays in the search result
+
+## Formatting
+
+- Backtick syntax elements that appear, inline, within a paragraph
+  - Cmdlet names `Verb-Noun`
+  - Variable `$counter`
+  - Syntactic examples `Verb-Noun -Parameter`
+  - File paths `C:\Program Files\PowerShell`, `/usr/bin/pwsh`
+  - URLs that are not meant to be clickable in the document
+- Use bold for property names, parameter values, parameter names, class names, module names, entity
+  names, object or type names
+  - Bold is used for semantic markup, not emphasis
+  - Bold - use asterisks `**`
+- Italic - use underscore `_`
+  - Only used for emphasis, not for semantic markup
+- Line breaks at 100 columns (or at 80 for **about_Topics**)
+- No hard tabs - use spaces only
+- No trailing spaces on lines
+
+### Headers
```

Comment:
I thought of another to add here.  That headers should "gradually step down".  ie... no going straight down from H2 to H4.

Reply:
added